addons/knative/values.yaml

```diff
@@ -0,0 +1,5 @@
+# Specify the repository name
+name: knative
+
+# Specify the URL where the repository will be served
+url: file:///.
```

Comment:
I don't believe we need the values here.

Reply:
You are right, removed.